spec/models/user_spec.rb

```diff
@@ -65,19 +65,6 @@
     user.confirm
   end
 
-  it "requires region_id for a delegate" do
-    delegate = FactoryBot.create :delegate
-    region_id = delegate.region_id
-    user = FactoryBot.create :user
-
-    delegate.region_id = user.region_id
-    expect(delegate).to be_invalid_with_errors(region_id: ["can't be blank"])
-
-    delegate.region_id = region_id
-    user.update(delegate_status: "delegate", region_id: region_id)
-    expect(delegate).to be_valid
-  end
-
```

Comment:
This unit test is no longer required because the region_id column is no longer updated, and will be removed in another PR once this PR merges. In the intermediate period, the region_id is not updated even if a delegate role is added.